js/flightlog_parser.js

```diff
@@ -1108,8 +1141,8 @@ var FlightLogParser = function(logData) {
             predictor = frameDef.predictor,
             encoding = frameDef.encoding,
             values = new Array(8),
-            i, j, groupCount;
-
+            i, j, groupCount, updateCurrent;
```

Comment:
Try not adding new vars - please declare / initialize at block scope.

Reply:
The function where this i, j variables are defined use them in loops and after the loops, so in this case I recommend keeping them. They are defined that way because of the logic in this function.